shared/src/iosMain/kotlin/by/audiobooks/mob/di/KoinIOS.kt

```diff
@@ -17,14 +16,14 @@ actual val platformModule = module {
     }
 }
 
-@kotlinx.cinterop.BetaInteropApi
-fun Koin.get(objCClass: ObjCClass): Any {
-    val kClazz = getOriginalKotlinClass(objCClass)!!
-    return get(kClazz, null, null)
-}
-
-@kotlinx.cinterop.BetaInteropApi
-fun Koin.get(objCClass: ObjCClass, qualifier: Qualifier?, parameter: Any): Any {
-    val kClazz = getOriginalKotlinClass(objCClass)!!
-    return get(kClazz, qualifier) { parametersOf(parameter) }
-}
+fun getRepositoryClient(): Repository {
+    return RepositoryImpl(
+        dbHelper = DatabaseHelper(
+            sqlDriver = NativeSqliteDriver(
+                AudiobooksByDB.Schema,
+                "audiobooks-by.db"
+            )
+        ),
+        siteApi = SiteApi()
```

Comment:
Yesterday I merged SearchClient, so probably another argument will be required here to build `RepositoryImpl`

Reply:
Yep:
<img width="1239" alt="image" src="https://github.com/user-attachments/assets/4820ea38-a37c-4260-8ad7-07a6dbff500e">

Reply:
fixed